[JS] feat: `Azure OpenAI On Your Data` - Azure AI Search support (#1426)
microsoft/teams-ai: js/samples/08.datasource.azureOpenAI/README.md

# Azure OpenAI On Your Data

> The following is a conversational bot that uses the Azure OpenAI Chat Completions API and more specifically the `Azure OpenAI On Your Data` feature to facilitate RAG (retrieval augmentation) from Azure data sources, more specifically Azure AI Search.

Reviewer: ```suggestion
The following is a conversational bot that uses the Azure OpenAI Chat Completions API `Azure OpenAI on Your Data` feature to facilitate RAG (retrieval augmentation) using Azure AI Search as the Azure data source.
```
